docs/src/appendix/cast/account/create.md

```diff
@@ -0,0 +1,24 @@
+# `create`
+Create all data required for account deployment.
```

Comment:
This is fine actually, but I wonder if it wouldn't be better if we phrase it like:

`Prepare all prerequisites for account deployment.`

what do you think?